test/runner.js

```diff
@@ -10,7 +10,7 @@ let port;
 
 // Should be cmd-line args at some point
 const debugMode = true;
-const skipBuild = false;
+const skipBuild = true;
```

Comment:
Changed this temporarily so I don't have to wait a minute before each test run. Will revert before making the full PR and will be changed to a CLI arg once the service-tooling stuff is in.

Reply:
This is now changed back.